src/server/routes.ts

```diff
@@ -13,4 +13,19 @@ router.get('/users', async (_req, res) => {
   res.json(users);
 });
 
+router.post('/users/login', async (req, res, next) => {
+  try {
+    const { username, password } = req.body;
+    const user = await readUser({ username, password });
+    if (!user) {
+      res.status(404).send('User not found');
+      return;
+    }
+
```

Comment:
It would be awesome to send a cookie to the client to identify the user in upcoming requests

Reply:
You're right. Is done.